src/Runners/PHPUnit/WrapperRunner.php

```diff
@@ -24,6 +24,12 @@ final class WrapperRunner extends BaseRunner
 
     protected function beforeLoadChecks(): void
     {
+        if ($this->options->teamcity()) {
+            throw new InvalidArgumentException(
+                'The `teamcity` option is not supported yet in the WrapperRunner.'
```

Comment:
May I ask you why?

Reply:
Now, `--log-teamcity` option supports it